EVEStandard/API/Fittings.cs

```diff
@@ -42,15 +42,15 @@ public async Task DeleteFittingV1Async(AuthDTO auth, long fittingId)
         /// <param name="auth">The <see cref="AuthDTO"/> object.</param>
         /// <param name="ifNoneMatch">ETag from a previous request. A 304 will be returned if this matches the current ETag.</param>
         /// <returns><see cref="ESIModelDTO{T}"/> containing a list of fittings.</returns>
-        public async Task<ESIModelDTO<List<CharacterFitting>>> GetFittingsV1Async(AuthDTO auth, string ifNoneMatch = null)
+        public async Task<ESIModelDTO<List<ShipFitting>>> GetFittingsV2Async(AuthDTO auth, string ifNoneMatch = null)
```

Comment:
CharacterFitting for the model to ShipFitting? Could you explain please?

Reply:
It's the same model. I believe two models were created by swagger tool: one for POST and one for GET. Additionally, there seems to be quite a few models in the library that are not used at all.
It's possible that at some point the two models could end up not being the same, but they are right now. If we were to distinguish between Request and Response models we should probably name them in a way that would indicate that. TL;DR; There are spare, unused and duplicate models in the library. This is an attempt to start tidying it up.